src/Node/BreakNode.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Node;
+
+use Twig\Compiler;
+
+final class BreakNode extends Node
+{
+    private int $level;
+
+    public function __construct(int $level)
+    {
+        parent::__construct([], [], 0, null);
+
+        $this->level = $level;
+    }
+
+    public function compile(Compiler $compiler): void
+    {
+        $compiler->write(
```

Comment:
please add the call to `->addDebugInfo($this)` at the beginning of the compilation, to have better debug info in the generated code (this BreakNode is a statement-like node)

Reply:
thanks, done!